server/pxf-jdbc/src/test/java/org/greenplum/pxf/plugins/jdbc/utils/DbProductTest.java

```diff
@@ -105,6 +128,21 @@ public void testOracleTimestamps() {
         }
     }
 
+    @Test
+    public void testOracleBuildEnvQuery() {
+        final Properties EXPECTED = new Properties();
```

Comment:
I think these tests should be simplified to the following
```suggestion
        DbProduct dbProduct = DbProduct.getDbProduct(DB_NAME_ORACLE);
        assertEquals("ALTER SESSION SET key = value;", dbProduct.buildEnvQuery(entry));
```

Reply:
I would prefer these tests to stay unchanged. 

There is [a cycle](https://github.com/greenplum-db/pxf/pull/118/files#diff-698f3ad022ae10ac10c6618a799970ceR140) over `ENVS` (which may contain multiple values). In addition, due to the fact that `dbProduct.buildEnvQuery()` receives `Entry` (which can only be retrieved from `Properties`), some kind of [iteration](https://github.com/greenplum-db/pxf/pull/118/files#diff-698f3ad022ae10ac10c6618a799970ceR138) would still be required even in case we move to the proposed simplified comparison.